static/css/styles.css

```diff
@@ -0,0 +1,17 @@
+body {
+    font-family: Roboto, Arial, sans-serif;
+    font-size: 18px;
+    margin: 50px;
+    vertical-align: middle;
+}
+
+.sidebar li {
+    font-size: 1.2rem;
+    padding: 0.5em 0;
+    list-style: none;
+    border-bottom: 1px solid lightgray;
+}
+
+.container ul {
+    list-style: square;
+}
```

Comment:
add blank lines to the end of the files where you don't have it

Reply:
If no blank line has been added to the file, GitHub marks it with a red circle?)

Reply:
yep